docs/MediaCodecTests.md

```diff
@@ -33,58 +39,95 @@ Below is a diagram on how input buffers turn into output buffers.
+Sound-checker creates a sine wave of the certain sample rate and passes the PCM data through an
+encoder and decoder. The Harmonic Analyzer calculates the fundamental bin and evaluates the
+results. Each analysis comprises 1024 bins and gives a result for Total Harmonic Distortion (THD),
+Total Harmonic Distortion + Noise (THD + N), Signal to Noise (SNR), and Peak Amplitude in decibels
+(nPeak). Ideally, we would want lower THD and THD+N values and higher SNR and nPeak values.
 
 ![MediaCodec sine wave tests](images/media-codec-sine-wave-tests.png "MediaCodec sine wave tests")
 
-A waveform of the final signal is also provided. This should ideally be a sine wave. The graph of the frequency bins is shown with a gray background. The left bins represent lower frequencies and the right bins represent higher frequencies. This is ideally a single vertical line, signaling that the signal is all within one frequency bin.
+A waveform of the final signal is also provided. This should ideally be a sine wave. The graph of
+the frequency bins is shown with a gray background. The left bins represent lower frequencies and
+the right bins represent higher frequencies. This is ideally a single vertical line, signaling
+that the signal is all within one frequency bin.
 
 ## Sine Sweep Tests
 
-Instead of encoding a sine wave, the sine sweep tests create a sine sweep by modulating the frequency of the sine wave. The variable phase increment will result in a tone that increases and decreases in pitch.
+Instead of encoding a sine wave, the sine sweep tests create a sine sweep by modulating the
+frequency of the sine wave. The variable phase increment will result in a tone that increases and
+decreases in pitch.
 
 ![MediaCodec sine sweep tests](images/media-codec-sine-sweep-tests.png "MediaCodec sine sweep tests")
 
-THD and other metrics no longer apply for sine sweep tests as the result should have multiple frequency bins instead of one. Instead, we can look at a spectrogram to see which frequencies the sine wave is in over time.
+THD and other metrics no longer apply for sine sweep tests as the result should have multiple
+frequency bins instead of one. Instead, we can look at a spectrogram to see which frequencies the
+sine wave is in over time.
 
-For a spectrogram, each vertical bar is the density of frequencies for a particular time frame. So instead of frequency bins being in one graph in the sine wave graph, the frequency bins are mapped to one line, with higher frequencies being higher. We should expect most data to be within the correct signal bin, resulting in an appearance of hills.
+For a spectrogram, each vertical bar is the density of frequencies for a particular time frame. So
+instead of frequency bins being in one graph in the sine wave graph, the frequency bins are mapped
+to one line, with higher frequencies being higher. We should expect most data to be within the
+correct signal bin, resulting in an appearance of hills.
```

Comment:
This "hills" thing confused me. I was thinking of looking down on a topgraphic map!

Maybe describe the difference between a good and bad spectrogram. Smooth narrow line versus a line that is smeared vertically.